python/PDFs/basic/PolynomialPdf.cpp

```diff
@@ -19,7 +19,7 @@ void init_PolynomialPdf(py::module &m) {
              "lowestDegree"_a = 0)
         .def(py::init<std::string, Observable, std::vector<Variable>, unsigned int>(),
              PolynomialPdf_docs.c_str(),
-             "nane"_a,
+             "name"_a,
```

Comment:
I'm not going to tag @AdamChainz every time code spell catches a real API error, but maybe once or twice ;) https://twitter.com/AdamChainz/status/1459076392801517600?s=20

(This is the only PDF with `nane="..."` instead of `name="..."` out of several dozen)

Reply:
haha cool :)